apps/web/pages/v2/workflows/[workflow].tsx

```diff
@@ -0,0 +1,28 @@
+import { GetStaticPaths, GetStaticProps } from "next";
```

Comment:
Copied file, no changes

Reply:
Will add some deprecation comments to the v1 file